src/scripting/V3d.hpp

```diff
@@ -122,6 +122,8 @@ public slots:
 	//! Formats a comma-separated string in angle brackets
 	//! @todo currently this seems to to nothing. JS print output shows "V3f(address)". Use toVec3f() to show the values formatted by Vec3f's method.
 	Q_INVOKABLE QString toString() const {return QString("[%1, %2, %3]").arg(m_x).arg(m_y).arg(m_z);}
+	//! Formats a hex string
```

Comment:
Please provide better documentation. What is this for, HTML color formatting?

Reply:
Yes, I've just added missing function (copy + paste + fix types)